kimchi/src/circuits/polynomials/rot.rs

```diff
@@ -56,7 +56,7 @@ impl<F: PrimeField + SquareRootField> CircuitGate<F> {
             CircuitGate {
                 typ: GateType::RangeCheck0,
                 wires: Wire::for_row(new_row + 1),
-                coeffs: vec![],
+                coeffs: vec![F::zero()],
```

Comment:
`coeffs: vec![]` is the same here

Reply:
It's not; the test fails without it.

Reply:
ah interesting, there must be some places that don't handle the coeffs gracefully : o